Tests/Packet++Test/Tests/TelnetTests.cpp

```diff
@@ -24,47 +24,51 @@ PTF_TEST_CASE(TelnetCommandParsingTests)
 	PTF_ASSERT_EQUAL(telnetLayer->getDataAsString(), "");
 	PTF_ASSERT_EQUAL(telnetLayer->getTotalNumberOfCommands(), 8);
 
-	PTF_ASSERT_EQUAL(telnetLayer->getNumberOfCommands(pcpp::TelnetLayer::WillPerform), 1);
-	PTF_ASSERT_EQUAL(telnetLayer->getNumberOfCommands(pcpp::TelnetLayer::DoPerform), 5);
-	PTF_ASSERT_EQUAL(telnetLayer->getNumberOfCommands(pcpp::TelnetLayer::SubnegotiationEnd), 1);
+	PTF_ASSERT_EQUAL(telnetLayer->getNumberOfCommands(pcpp::TelnetLayer::TelnetCommand::WillPerform), 1);
+	PTF_ASSERT_EQUAL(telnetLayer->getNumberOfCommands(pcpp::TelnetLayer::TelnetCommand::DoPerform), 5);
+	PTF_ASSERT_EQUAL(telnetLayer->getNumberOfCommands(pcpp::TelnetLayer::TelnetCommand::SubnegotiationEnd), 1);
 
-	PTF_ASSERT_EQUAL(telnetLayer->getFirstCommand(), pcpp::TelnetLayer::WillPerform);
+	PTF_ASSERT_TRUE(telnetLayer->getFirstCommand() == pcpp::TelnetLayer::TelnetCommand::WillPerform);
 
-	PTF_ASSERT_EQUAL(telnetLayer->getOption(pcpp::TelnetLayer::WillPerform), pcpp::TelnetLayer::SuppressGoAhead);
-	PTF_ASSERT_EQUAL(telnetLayer->getOption(pcpp::TelnetLayer::DoPerform), pcpp::TelnetLayer::TerminalType);
-	PTF_ASSERT_EQUAL(telnetLayer->getOption(pcpp::TelnetLayer::AreYouThere), pcpp::TelnetLayer::TelnetOptionNoOption);
+	PTF_ASSERT_TRUE(telnetLayer->getOption(pcpp::TelnetLayer::TelnetCommand::WillPerform) ==
+					pcpp::TelnetLayer::TelnetOption::SuppressGoAhead);
+	PTF_ASSERT_TRUE(telnetLayer->getOption(pcpp::TelnetLayer::TelnetCommand::DoPerform) ==
+					pcpp::TelnetLayer::TelnetOption::TerminalType);
+	PTF_ASSERT_TRUE(telnetLayer->getOption(pcpp::TelnetLayer::TelnetCommand::AreYouThere) ==
+					pcpp::TelnetLayer::TelnetOption::TelnetOptionNoOption);
```

Comment:
Instead of `PTF_ASSERT_TRUE()` you can use:
```cpp
PTF_ASSERT_EQUAL(..., ..., enumclass);
```

Same applies to more changes below...